packages/labs/src/frameworks/react/create-component.ts

```diff
@@ -0,0 +1,216 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as ReactModule from 'react';
+
+const reservedReactProperties = new Set([
+  'children',
+  'localName',
+  'ref',
+  // TODO(sorvell): why are the properties below included?
+  'style',
+  'className',
+]);
+
+const listenedEvents: WeakMap<
+  Element,
+  Map<string, EventListenerObject>
+> = new WeakMap();
+
+const setEvent = (
+  node: Element,
+  event: string,
+  value: (event?: Event) => void
+) => {
+  let events = listenedEvents.get(node);
+  if (events === undefined) {
+    listenedEvents.set(node, (events = new Map()));
+  }
+  let handler = events.get(event);
+  if (value !== undefined) {
+    // If necessary, add listener and track handler
+    if (handler === undefined) {
+      events.set(event, (handler = {handleEvent: value}));
+      node.addEventListener(event, handler);
+      // Otherwise just update the listener with new value
+    } else {
+      handler.handleEvent = value;
+    }
+    // Remove listener if one exists and value is undefined
+  } else if (handler !== undefined) {
+    events.delete(event);
+    node.removeEventListener(event, handler);
+  }
+};
+
+const setProperty = <E extends Element, T>(
+  node: E,
+  name: string,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any,
```

Comment:
`any` -> `unknown`

Reply:
looks like `old` got changed to `unknown`, but not `value`

Reply:
That results in `Type 'unknown' is not assignable to type 'E[keyof E]'.`

Reply:
Right, but that's a better case than an any and a linter exception since it spells out the intention.

You could also try to pass the merged props type here and make `<..., K extends keyof P>(..., name: K, value: P[K]` but that will likely push a cast to the caller. This is all very dynamic and internal at this point in the calls, so I don't know if it buys much, but in general it's at least better to avoid `any` by using `unknown` and a specific cast.

Reply:
Done.